src/lib/utils/application-commands/ApplicationCommandRegistries.ts

```diff
@@ -46,6 +47,20 @@ export function getDefaultBehaviorWhenNotIdentical() {
 	return defaultBehaviorWhenNotIdentical;
 }
 
+/**
+ * Sets the default guild ids for registering commands. This overwrites any usages of `guildIds` in the regular
```

Comment:
This should not override manual guildIds passed in, it should strictly be the default if its missing